docs/src/components/home/sections/AmplifySection.tsx

```diff
@@ -0,0 +1,77 @@
+import { useRef } from 'react';
+import classNames from 'classnames';
+import { Heading, Link, View, Flex } from '@aws-amplify/ui-react';
+import { AmplifyIcon, ServerIcon, LibraryIcon } from '@/components/Icons';
+import { useIntersectionObserver } from '@/components/useIntersection';
+import { HomeFeatureCard } from '../HomeFeatureCard';
+
+export const AmplifySection = ({ platform }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const entry = useIntersectionObserver(ref, {
+    threshold: 0.125,
+    freezeOnceVisible: true,
+  });
+  const isVisible = !!entry?.isIntersecting;
+
+  return (
+    <View
+      ref={ref}
+      as="section"
+      testId="docs-home-section"
+      className={classNames(
+        'docs-home-section',
+        'fade-in',
+        isVisible && 'shown'
+      )}
+    >
+      <Flex direction="column" className="docs-home-subsection--thin">
+        <Heading level={2}>
+          Better <strong>together</strong> with AWS Amplify
+        </Heading>
+
+        <HomeFeatureCard
+          title="Build your application visually in Amplify Studio"
+          icon={AmplifyIcon}
+        >
+          Deploy and manage your app backend with a GraphQL API, User login,
+          File storage and more in minutes. Start small and build to the full
+          scale of AWS!
+          {platform === 'react'
+            ? ' Build components visually in Figma, bind them to your data, and use them in your codebase.'
+            : ''}
+          <br />
+          <Link href="https://docs.amplify.aws/console/">
+            Learn more about Amplify Studio
+          </Link>
+        </HomeFeatureCard>
+        <HomeFeatureCard
+          title="Connect your UI to your backend easily with Amplify libraries."
```

Comment:
Just a thought: I read/heard something that recommended avoiding words like "easily" for this kind of copywriting.